Creational/README.rst

```diff
@@ -6,7 +7,7 @@ suitable to the situation. The basic form of object creation could result in
 design problems or added complexity to the design. Creational design patterns
 solve this problem by somehow controlling this object creation.
 
-* [AbstractFactory](AbstractFactory) [:notebook:](http://en.wikipedia.org/wiki/Abstract_factory_pattern)
+* :doc:`AbstractFactory <AbstractFactory/README>` `Wikipedia <http://en.wikipedia.org/wiki/Abstract_factory_pattern>`_
```

Comment:
@domnikl I didn't think there was much interest, so I put it lower down my list. I can look into it again.